tests/providers_test.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2015 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for perfkitbenchmarker.providers"""
+
+import unittest
+
+from perfkitbenchmarker import benchmark_spec
+from perfkitbenchmarker import providers
+
+
+class LoadProvidersTestCase(unittest.TestCase):
+
+  def testLoadAllProviders(self):
+    for cloud in benchmark_spec.VALID_CLOUDS:
+      providers.LoadProvider(cloud.lower())
+
+  def testLoadInvalidProvider(self):
+    with self.assertRaises(ImportError):
+      providers.LoadProvider('InvalidCloud')
+
+
+if __name__ == '__main__':
```

Comment:
You don't need the `unittest.main()` call with our testing framework, although as far as I know there's no harm in it.